Dockerfile

```diff
@@ -47,7 +51,7 @@ RUN /bin/sh /app/docker/utility/install_sfdx.sh
 # installing python related dependencies with pip
 COPY ./requirements /app/requirements
 RUN pip install --no-cache --upgrade pip
-RUN if [ "${BUILD_ENV}" = "production" ] ; then pip install --no-cache -r /app/requirements/production.txt ; else pip install --no-cache -r /app/requirements/local.txt ; fi
+RUN pip install --no-cache -r /app/requirements/$BUILD_ENV.txt 
```

Comment:
Nice simplification.

I just noticed, I think the flag for pip is supposed to be `--no-cache-dir` rather than `--no-cache`